infrasec/tutorials/atlantis.md

```diff
@@ -0,0 +1,418 @@
+
+### Add & validate a certificate for Atlantis
+
+We'll need to add a new certificate to ACM, which [manages our certificates for us](https://docs.aws.amazon.com/acm/latest/userguide/acm-overview.html). There are two steps: adding the certificate, and validating the certificate. We can't create **_and_** validate Route53 records in Govcloud. For a more thorough explanation, see our Engineering Playbook documentation on [ACM in GovCloud](https://github.com/trussworks/Engineering-Playbook/blob/23cad804d1390fc5def7a9f7b41d44db88b902a8/infrasec/aws/govcloud/gov-acm.md). To reiterate briefly here, we'll need to:
+
+1. Create the new certificate in GovCloud using the [`aws_acm_certificate` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate)
+2. Export the values as an output in the `output.tf` file like so:
+
+    ```hcl
+    output "atlantis_cert_validation" {
+      description = "Map of data for atlantis. cert validation"
+      value       = aws_acm_certificate.acm_atlantis_external.domain_validation_options
+    }
+    ```
+
+    Once we've merged the PR, we'll have the cert validation output value to validate the certificate in the next step.
+
+3. Validate the Route53 DNS records in the Commercial account using the [`aws_acm_certificate_validation` resource](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/acm_certificate_validation).
+
+    We'll need to find the `arn` to plug into the [atlantis module](https://registry.terraform.io/modules/terraform-aws-modules/atlantis/aws/latest) we'll call in the next step. We can find this in the console's Certificate Manager after merging the PR to add the certificate.
+
+    <img src="https://github.com/trussworks/Engineering-Playbook/blob/3efe6ea02ed010f3db2c07921c5c8acc60406b84/infrasec/tutorials/images/atlantis_acm1.png" width="450">
+
+### Set up Docker 🐋
+
+For any project not requiring a specific type of docker implementation, we can simply [pass in the `trussworks-atlantis-ecs-image` image](https://github.com/trussworks/trussworks-atlantis-ecs-image) when we make the module call in [Step 3](#step-3-call-the-atlantis-module-).
+
+If we do not set the `atlantis_image` variable, we'll find `atlantis:latest` is used by default. This default is not recognized as updated when a new "latest" is released due to the word remaining unchanged. Therefore, we recommend passing in a numbered version of the Atlantis docker image to the `atlantis_image` var.
```

Comment:
This isn't really about setting up docker, it's about setting the image for Fargate to use. There's two pieces of info you want to communicate here -- what image to use (the truss one? or your own for your project?) and that you want to set specific image, not `latest`.

Reply:
Yeah this was copy/paste from what was already in the doc. I can clean it up a little.